README.md

```diff
@@ -32,4 +32,4 @@ $ foreman start
 
 The original [Hemingway App](http://www.hemingwayapp.com/).
 
-Photo by [Leon Bliss](https://www.flickr.com/photos/leonbiss/) via [Unsplash](http://unsplash.com/).
\ No newline at end of file
+Photo by [Leon Bliss](https://www.flickr.com/photos/leonbiss/) via [Unsplash](http://unsplash.com/).
```

Comment:
You're right that it's some issue with my text editor (Atom). I tried troubleshooting it again, but all of the whitespace settings I could find were set appropriately. I'm not sure what's causing it.

In any case, I made this pull request just to get some screenshots for a blog post. Feel free to ignore it.